test/e2e/common.go

```diff
@@ -92,6 +92,7 @@ const (
 	aksClusterNameSuffix            = "aks"
 	flatcarCAPICommunityGallery     = "flatcar4capi-742ef0cb-dcaa-4ecb-9cb0-bfd2e43dccc0"
 	defaultNamespace                = "default"
+	AzureCNIv1Manifest              = "AZURE_CNI_V1_MANIFEST_PATH"
```

Comment:
Azure CNI "v1" is how we've been referring to the legacy Azure CNI implementation to differentiate it from "v2", however, is this how it's also referred to in the Azure CNI repo and docs? If not we should probably stick with calling it just "Azure CNI"

Reply:
Looks like azure container networking team also refers to this Manifest as `v1` installer. Reference: https://github.com/Azure/azure-container-networking/blob/master/hack/manifests/cni-installer-v1.yaml

Maybe we keep it as is?

Reply:
Sounds good